src/lib/Controller/Makefile

```diff
@@ -33,38 +33,24 @@ CFLAGS      = -fPIC -Wall
 INCLUDE     = -I$(WEBOTS_PATH)/include/controller/c -I$(WEBOTS_PATH)/include
 LD_FLAGS    = -shared -Wl,-soname,libController.so
 STATIC_LIBS =
-ifeq ($(UBUNTU_VERSION), 16.04)
-LIB_PNG      = png12
-else
-LIB_PNG      = png16
-endif
-SHARED_LIBS = -ltiff -lm -ljpeg -lpthread -l$(LIB_PNG) -ldl -llzma -lrt
+SHARED_LIBS = -lm -lpthread -ldl -llzma -lrt
 TARGET      = $(WEBOTS_CONTROLLER_LIB_PATH)/libController.so
 endif
 
 ifeq ($(OSTYPE),windows)
 CFLAGS      = -Wall -mwindows
 INCLUDE     = -I$(WEBOTS_PATH)/include/controller/c -I$(WEBOTS_PATH)/include -I/mingw64/include
 LD_FLAGS    = -shared -mwindows --def Controller.def
-STATIC_LIBS = -static -lpng -lz -ljpeg -ltiff -llzma -lzstd
+STATIC_LIBS = -static -lz -llzma -lzstd
```

Comment:
I believe these libraries are dependencies of png, jpeg or tiff and we should remove them as well.

Reply:
You're right I tested and it seems we don't need to link with them anymore (but we still need to include them in the package as Webots still relies on them).